From 0efaefce4e95a3331550329c0078b2fb38b3ff1f Mon Sep 17 00:00:00 2001
From: Maxwell Doose <maxwell@maxwelld.cc>
Date: Fri, 31 Jul 2026 16:39:11 -0500
Subject: [PATCH 10661/15333] iio: Update email for Maxwell Doose

I'm transitioning to a new email, add mapping and update MAINTAINERS
entries to keep email up to date.

Signed-off-by: Maxwell Doose <maxwell@maxwelld.cc>
Signed-off-by: Jonathan Cameron <jonathan.cameron@oss.qualcomm.com>
---
 .mailmap                                                        | 1 +
 .../devicetree/bindings/iio/chemical/sensirion,scd30.yaml       | 2 +-
 MAINTAINERS                                                     | 2 +-
 3 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/.mailmap b/.mailmap
index 2b09883a19d6..5261076e32b4 100644
--- a/.mailmap
+++ b/.mailmap
@@ -589,6 +589,7 @@ Maxim Mikityanskiy <maxtram95@gmail.com> <maximmi@nvidia.com>
 Maxime Ripard <mripard@kernel.org> <maxime@cerno.tech>
 Maxime Ripard <mripard@kernel.org> <maxime.ripard@bootlin.com>
 Maxime Ripard <mripard@kernel.org> <maxime.ripard@free-electrons.com>
+Maxwell Doose <maxwell@maxwelld.cc> <m32285159@gmail.com>
 Maya Erez <quic_merez@quicinc.com> <merez@codeaurora.org>
 Mayuresh Janorkar <mayur@ti.com>
 Md Sadre Alam <quic_mdalam@quicinc.com> <mdalam@codeaurora.org>
diff --git a/Documentation/devicetree/bindings/iio/chemical/sensirion,scd30.yaml b/Documentation/devicetree/bindings/iio/chemical/sensirion,scd30.yaml
index 25291bda1a8b..ea0e72f2a690 100644
--- a/Documentation/devicetree/bindings/iio/chemical/sensirion,scd30.yaml
+++ b/Documentation/devicetree/bindings/iio/chemical/sensirion,scd30.yaml
@@ -7,7 +7,7 @@ $schema: http://devicetree.org/meta-schemas/core.yaml#
 title: Sensirion SCD30 carbon dioxide sensor
 
 maintainers:
-  - Maxwell Doose <m32285159@gmail.com>
+  - Maxwell Doose <maxwell@maxwelld.cc>
   - Tomasz Duszynski <tduszyns@gmail.com>
 
 description: |
diff --git a/MAINTAINERS b/MAINTAINERS
index bd5cfcb7d887..e84cd5d08e86 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -24564,7 +24564,7 @@ F:	Documentation/devicetree/bindings/iio/chemical/senseair,sunrise.yaml
 F:	drivers/iio/chemical/sunrise_co2.c
 
 SENSIRION SCD30 CARBON DIOXIDE SENSOR DRIVER
-M:	Maxwell Doose <m32285159@gmail.com>
+M:	Maxwell Doose <maxwell@maxwelld.cc>
 M:	Tomasz Duszynski <tduszyns@gmail.com>
 S:	Maintained
 F:	Documentation/devicetree/bindings/iio/chemical/sensirion,scd30.yaml
-- 
2.55.0

